.ocamlformat

```diff
@@ -0,0 +1,27 @@
+break-infix = fit-or-vertical
```

Comment:
These are my personal preferences, lifted from https://github.com/anmonteiro/piaf/blob/master/.ocamlformat

Happy to bikeshed formatting.